examples/ephemeral-spark/ephemeral-spark.tf

```diff
@@ -1,17 +1,13 @@
 # Ephemeral Spark cluster
 module "ephemeral-spark-sgs" {
-  source                        = "git::git@github.com:Datatamer/terraform-aws-emr.git//modules/aws-emr-sgs?ref=3.0.0"
-  applications                  = ["Spark"]
-  vpc_id                        = var.vpc_id
-  emr_managed_master_sg_name    = "${var.name_prefix}-EMR-Spark-Master"
-  emr_managed_core_sg_name      = "${var.name_prefix}-EMR-Spark-Core"
-  emr_additional_master_sg_name = "${var.name_prefix}-EMR-Spark-Additional-Master"
-  emr_additional_core_sg_name   = "${var.name_prefix}-EMR-Spark-Additional-Core"
-  emr_service_access_sg_name    = "${var.name_prefix}-EMR-Spark-Service-Access"
+  source              = "git::git@github.com:Datatamer/terraform-aws-emr.git//modules/aws-emr-sgs?ref=6.0.0"
```

Comment:
Not really, removing these variables is a reflect of using tag 6.x.x for EMR. Because now those SGs are created with SG module and received as IDs in here.

Reply:
Not this line specifically but in other places yes, as I'm already using tag 6.1.0. You're asking maybe because I forgot updating this to 6.1.0? Both should work as I'm not modifying anything in this submodule.